tests/rule-types/rule-types.rules

```diff
@@ -9,10 +9,18 @@ alert tcp-pkt any any -> any any (msg:"tcp-pkt, anchored content"; content:"abc"
 alert tcp any any -> any any (msg:"tcp, no content"; sid:301;)
 alert tcp any any -> any any (msg:"tcp, simple content"; content:"abc"; sid:302;)
 alert tcp any any -> any any (msg:"tcp, anchored content"; content:"abc"; startswith; sid:303;)
-alert tcp !192.168.0.1 any -> any any (msg:"tcp, negated IP address"; sid:304;)
+alert tcp !192.168.0.1 any -> any any (msg:"tcp, has negated IP address"; sid:304;)
+alert tcp [10.0.0.0/8,!10.10.10.10] any -> [10.0.0.0/8,!10.10.10.10] any (msg:"tcp, has negated IP address"; sid:305;)
```

Comment:
Should we split this test in one test per rule type ?

Reply:
Could be a good idea, except for cases where we want to see the interaction of keywords and rule types or states.